modules/data-entry/src/main/frontend/src/dataHomepage/PrintButton.jsx

```diff
@@ -59,6 +59,24 @@ function PrintButton(props) {
 
   const [ open, setOpen ] = useState(false);
 
+  // Prevent browser to open print dialog on user ctrl+P keydown and force to go through the custom print preview
+  useEffect(() => {
+    // subscribe event
+    window.addEventListener("keydown", handleOnPrintKeydown);
+    return () => {
+      // unsubscribe event
+      document.removeEventListener("keydown", handleOnPrintKeydown);
+    };
+  });
```

Comment:
I think this should solve the double dialog open:

```suggestion
  }, []);
```

Reply:
`ResourceHeader` duplicates `PrintButton`, so there are actually 2 previews that are getting open.

Reply:
Nope. While this change should indeed be made to avoid adding and removing the listener unnecessarily with state changes, the double print dialog is caused by the fact that the `ResourceHeader` component renders the menu twice: once next to the expanded title when the full header is displayed, and once in the collapsed version. We have to either render the menu on;y once or attach the listener only once.